clojure_v1/README.md

```diff
@@ -0,0 +1,29 @@
+# xservedbyfinder
+
+_a solution in Clojure_
+
+This solution lazily computes the distinct servers randomly selected and returned in *X-Served-By* headers. It demonstrates two interesting clojure idioms: lazy sequences and tail recursion. The function ``sample-attempts`` encapsulates the logic that determines how many attempts you make at finding a new value before you decide you have enough confidence to stop hunting for another.
+
+## Usage
+
+To run this, download and install [leiningen](http://leiningen.org). Then go to the folder containing this README and type
+
+    lein run
+    
```

Comment:
Is it possible to add the source to the Docker image and then install leiningen inside the container, so that it can execute lein run / lein uberjar?